### PR TITLE
Replace private api key mention

### LINKED ISSUE #239: Change the naming of `Private Key`
Since v0.25.0 the term `private` key is not used anywhere. 
See [gatsby section on API keys](https://github.com/meilisearch/gatsby-plugin-meilisearch/#-add-your-meilisearch-credentials). 

Another naming, or another formulation, should be used to tell user which key to use here: 

Screenshot 2022-02-10 at 11 35 33

Any suggestion @gmourier

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## What does this PR do?
Closes #239

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: cypress/integration/test-api-key-required.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ describe(`Test API key required`, () => {
   })
 
   it('Should find a text in modal requesting API key', () => {
-    cy.contains('Enter your private API key')
+    cy.contains('Enter your Default Admin API key')
```
Comment (bidoubiwa) [inline cypress/integration/test-api-key-required.js:17]:
```suggestion
    cy.contains('Enter your admin API key')
```

It does not need to be the default one

File: cypress/integration/test-no-api-key-required.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ describe(`Test no API key required`, () => {
   it('Should display a message telling that no api key is required', () => {
     cy.get('span').contains('Api Key').parent().click()
     cy.get('div[aria-label=settings-api-key]').within(() => {
-      cy.contains('Enter your private API key (optional)')
+      cy.contains('Enter your Default Admin API key (optional)')
```
Comment (bidoubiwa) [inline cypress/integration/test-no-api-key-required.js:25]:
```suggestion
      cy.contains('Enter your admin API key (optional)')
```

File: src/App.js [hunk 3]
```diff
@@ -208,7 +208,7 @@ const App = () => {
             </Body>
           </InstantSearch>
           <Modal
-            title={`Enter your private API key${
+            title={`Enter your Default Admin API key${
```
Comment (bidoubiwa) [inline src/App.js:211]:
```suggestion
            title={`Enter your admin API key${
```

File: src/components/Header/index.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ const ApiKey = ({ requireApiKeyToWork }) => {
         )}
       </DialogDisclosure>
       <Modal
-        title={`Enter your private API key${
+        title={`Enter your Default Admin API key${
```
Comment (bidoubiwa) [inline src/components/Header/index.js:45]:
```suggestion
        title={`Enter your admin API key${
```

File: src/components/ApiKeyModalContent.js [hunk 5]
```diff
@@ -63,8 +64,15 @@ const ApiKeyModalContent = ({ closeModal }) => {
       </Box>
       <Box position="relative">
         <Typography variant="typo11" my={3} color="gray.6" mt={2}>
-          At least a private API key is required for the dashboard to access the
-          indexes list.
+          At least one API key that allows <strong>indexes.get</strong>,{' '}
+          <strong>search</strong> and <strong>version</strong>{' '}
```
Comment (bidoubiwa) [inline src/components/ApiKeyModalContent.js:68]:
```suggestion
          At least one API key that gives permission to <strong>get the indexes</strong>,{' '}
          <strong>search</strong> and <strong> get the version of Meilisearch</strong>{' '}
```

File: src/components/ApiKeyModalContent.js [hunk 6]
```diff
@@ -63,8 +64,15 @@ const ApiKeyModalContent = ({ closeModal }) => {
...
+          <Link
+            href="https://docs.meilisearch.com/reference/api/keys.html#create-a-key"
+            target="_blank"
+          >
+            actions
```
Comment (bidoubiwa) [inline src/components/ApiKeyModalContent.js:73]:
```suggestion
```

File: src/components/ApiKeyModalContent.js [hunk 7]
```diff
@@ -63,8 +64,15 @@ const ApiKeyModalContent = ({ closeModal }) => {
...
+          </Link>{' '}
+          is required for the dashboard to access the indexes list.
```
Comment (bidoubiwa) [inline src/components/ApiKeyModalContent.js:75]: I think this is redundant with the phrase before.

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

**Comment by mdubus** [comment]
bors merge

**Pull request merged**